run_model-dep-full_limits_per_channel.sh

```diff
@@ -24,80 +27,69 @@ fi
 if [[ $MODEL == "mh125" ]]; then
     wsoutput="ws_mh125.root"
     modelfile="13,Run2017,mh125_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125} scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 elif [[ $MODEL == "mh125_lc" ]]; then
     wsoutput="ws_mh125_lc.root"
     modelfile="13,Run2017,mh125_lc_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125}(#tilde{#chi}) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 elif [[ $MODEL == "mh125_ls" ]]; then
     wsoutput="ws_mh125_ls.root"
     modelfile="13,Run2017,mh125_ls_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125}(#tilde{#tau}) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 elif [[ $MODEL == "mh125_align" ]]; then
     wsoutput="ws_mh125_align.root"
     modelfile="13,Run2017,mh125_align_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125} alignment scenario (h,H,A#rightarrow#tau#tau)"
-elif [[ $MODEL == "HH125" ]]; then
+    sub_analysis="sm-like-light"
+elif [[ $MODEL == "mHH125" ]]; then
     wsoutput="ws_mHH125.root"
     modelfile="13,Run2017,mHH125_13.root"
-    min_mass=150
-    max_mass=200
     scenario_label="M_{H}^{125} alignment scenario (h,H,A#rightarrow#tau#tau)"
-elif [[ $MODEL == "hm1125_cpv" ]]; then
-    #TODO this still need more modifications in the code
-    wsoutput="ws_hm1125_cpv.root"
-    modelfile="13,Run2017,hm1125_CPV_13.root"
-    analysis="mssm_vs_sm_CPV"
-    min_mass=130
-    max_mass=1500
+    sub_analysis="sm-like-heavy"
+elif [[ $MODEL == "mh1125_CPV" ]]; then
+    wsoutput="ws_mh1125_cpv.root"
+    modelfile="13,Run2017,mh1125_CPV_13.root"
     scenario_label="M_{h_1}^{125} (CPV) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="cpv"
 ### Negative mu scenarios #####
 elif [[ $MODEL == "mh125_muneg_1" ]]; then
     wsoutput="mh125_muneg_1.root"
     modelfile="13,Run2017,mh125_muneg_1_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125} (#mu = -1 TeV) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 elif [[ $MODEL == "mh125_muneg_2" ]]; then
     wsoutput="mh125_muneg_2.root"
     modelfile="13,Run2017,mh125_muneg_2_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125} (#mu = -2 TeV) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 elif [[ $MODEL == "mh125_muneg_3" ]]; then
     wsoutput="mh125_muneg_3.root"
     modelfile="13,Run2017,mh125_muneg_3_13.root"
-    min_mass=110
-    max_mass=3200
     scenario_label="M_{h}^{125} (#mu = -3 TeV) scenario (h,H,A#rightarrow#tau#tau)"
+    sub_analysis="sm-like-light"
 ### EFT scenarios #####
 elif [[ $MODEL == "mh125EFT" ]]; then
     wsoutput="mh125EFT.root"
     modelfile="13,Run2017,mh125EFT_13.root"
-    min_mass=110
-    max_mass=3200
+    y_min=1
```

Comment:
These variables are not used.

Reply:
y_min? I guess they were for plotting at some point

Reply:
removed